### ISSUE TITLE
Include new Glide64 Napalm plugin

### ISSUE THREAD

**Issue by richard42** [mupen64plus/mupen64plus-user-issues#197]
Originally reported on Google Code with ID 197

```

http://glide64.emuxhaven.net/index.php

This people are working in make their plugin widely portable.

I think can be a good idea work together with mupen64plus to include new
linux port in next version.

People apreciate a lot to play games nicely than have a lot of options, and
seems that this new versions has a lot of inprovements in gameplay.


```

Reported by `joseluisyelena` on 2009-01-08 16:05:09

**Comment by richard42** [comment]
```
I have read other issue with same. Please discard this issue
```

Reported by `joseluisyelena` on 2009-01-08 16:07:11

**Comment by richard42** [comment]
```
Duplicate of #170
```

Reported by `richard42g` on 2009-01-09 03:23:08
- **Status changed**: `Duplicate`
- **Merged into**: #170
